frontend/awx/resources/inventories/inventoryHostsPage/hooks/useInventoriesGroupsActions.tsx

```diff
@@ -0,0 +1,33 @@
+import { PencilAltIcon } from '@patternfly/react-icons';
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  IPageAction,
+  PageActionSelection,
+  PageActionType,
+  usePageNavigate,
+} from '../../../../../../framework';
+import { cannotEditResource } from '../../../../../common/utils/RBAChelpers';
+import { AwxRoute } from '../../../../main/AwxRoutes';
+import { InventoryGroup } from '../../../../interfaces/InventoryGroup';
+
+export function useInventoriesGroupsActions() {
+  const { t } = useTranslation();
+  const pageNavigate = usePageNavigate();
+
+  return useMemo<IPageAction<InventoryGroup>[]>(
+    () => [
+      {
+        type: PageActionType.Button,
+        selection: PageActionSelection.Single,
+        isPinned: true,
+        icon: PencilAltIcon,
+        label: t('Edit group'),
+        // Need to update routing to go to inventory group edit form when that will be implemented AAP-19476
```

Comment:
As a team we decided at the beginning of this project to try to avoid comments in the code, where possible, as they can get stale pretty quickly.  Here I think we have 2 options:

1. Go ahead and create the route, and have it implement the <PageNotImplemented> component. 
2. Remove this comment and do nothing.

Reply:
I'll go with option 2 and remove this comment